main/seccomp.c

```diff
@@ -0,0 +1,60 @@
+#include "seccomp-bpf.h"
+#include <signal.h>
+#include <stdio.h>
+#include <stddef.h>
+#include <string.h>
+#include <sys/prctl.h>
```

Comment:
Compile error on Cygwin and MinGW.
https://ci.appveyor.com/project/masatake/ctags/build/1.0.2153/job/d53g4g6ylncvha6g#L268
sys/prctl.h is not available on them.